compiler/rustc_target/src/callconv/riscv.rs

```diff
@@ -293,6 +293,10 @@ fn classify_arg<'a, Ty, C>(
         // Not touching this...
         return;
     }
+    if arg.layout.pass_indirectly_in_non_rustic_abis(cx) {
+        arg.make_indirect();
```

Comment:
Usually loongarch and riscv do exactly the same thing. Why does this one here not update `avail_gprs`?